### PR TITLE
Bobertz Vega Merge ?!?

### PR DESCRIPTION
**Pull request by ConnerBobertz**
please consider this merge.

### CODE REVIEW

File: src/main/java/com/csase/Student.java [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class Student {
     /*
      * Sets study planner and scanner
      */
```
Comment (IronPapas) [inline src/main/java/com/csase/Student.java:15]: This looks very exquisite!

Comment (ConnerBobertz) [reply src/main/java/com/csase/Student.java:15]: Im the best what can I say

Comment (IronPapas) [reply src/main/java/com/csase/Student.java:15]: You're so humble!

**Review by IronPapas — APPROVED** [review]
The code seems to be fixed now. Thank you very much!

**Pull request merged**